### PR TITLE
fix(dropdown):  update dropdown styles in Teams theme to match redlines

### PR DESCRIPTION
**Pull request by notandrew**
Updating the Dropdown styles to match redlines

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ### Fixes
 - Fix event lister leak in `FocusZone` @miroslavstastny ([#2227](https://github.com/microsoft/fluent-ui-react/pull/2227))
+- Fix `dropdown` styles to match Teams theme @notandrew ([#2172](https://github.com/microsoft/fluent-ui-react/pull/2172))
```
Comment (silviuaavram) [inline CHANGELOG.md]: `Dropdown` as in component.

File: packages/react/src/components/Dropdown/Dropdown.tsx [hunk 2]
```diff
@@ -53,6 +53,7 @@ export interface DropdownSlotClassNames {
   selectedItem: string
   selectedItems: string
   triggerButton: string
+  inverted: string
```
Comment (silviuaavram) [inline packages/react/src/components/Dropdown/Dropdown.tsx]: Don't need to add it as a slot.

Slots are used on complex components. For instance, a checkbox can have a label and a ticking box, so we will add those as slots to the checkbox component. Useful to apply styles.

Here `inverted` is just a prop, unless I am missing something.

File: packages/react/src/components/Dropdown/Dropdown.tsx [hunk 3]
```diff
@@ -1374,6 +1381,7 @@ Dropdown.slotClassNames = {
   selectedItem: `${Dropdown.className}__selecteditem`,
   selectedItems: `${Dropdown.className}__selected-items`,
   triggerButton: `${Dropdown.className}__trigger-button`,
+  inverted: '',
```
Comment (silviuaavram) [inline packages/react/src/components/Dropdown/Dropdown.tsx]: remove this as well

**Review by silviuaavram — APPROVED** [review]
I also saw https://screener.io/v2/states/BJRCORrtB.microsoft-fluent-ui-react/2172/1024x768/Chrome/popupexampleclosebuttonshorthandtsx-prepares-to-select-item-out-of-popup-0 (the one with the list of items visible).

The highlighted item has no changes, while the others have their header changed. Maybe it's nothing, but can you please double check?

Also left some comments, overall looks good! @mnajdova can you also take a look?

File: docs/src/examples/components/Dropdown/Types/index.tsx [hunk 4]
```diff
@@ -24,6 +24,11 @@ const Types = () => (
       description="A dropdown can be searchable and allow a multiple selection."
       examplePath="components/Dropdown/Types/DropdownExampleSearchMultiple"
     />
+    <ComponentExample
+      title="Inverted"
+      description="A dropdown can be on an lighter or darker background."
```
Comment (mnajdova) [inline docs/src/examples/components/Dropdown/Types/index.tsx]: Please make all descriptions of the inverted examples consistent.

File: packages/react/src/components/Dropdown/Dropdown.tsx (hunk 2 above)
Comment (mnajdova) [reply packages/react/src/components/Dropdown/Dropdown.tsx]: @silviuavram is right, this is just a regular prop, that should appear in the `DropdownProps` and `propTypes`, not in the `DropdownSlotClassNames`

File: packages/react/src/components/Dropdown/Dropdown.tsx [hunk 5]
```diff
@@ -122,6 +123,9 @@ export interface DropdownProps
   /** A dropdown can be formatted to appear inline next to other elements. */
   inline?: boolean
 
+  /** It can be on a dark background */
+  inverted?: boolean
```
Comment (mnajdova) [inline packages/react/src/components/Dropdown/Dropdown.tsx:127]: Consistent description with other inverted props. Also, please see how the other props descrption are defined in the form of `A dropdown can ....` 

Try to follow that format

File: packages/react/src/themes/teams-dark/componentVariables.ts [hunk 6]
```diff
@@ -15,5 +15,4 @@ export { default as Icon } from './components/Icon/iconVariables'
 export { default as Reaction } from './components/Reaction/reactionVariables'
 export { default as Alert } from './components/Alert/alertVariables'
 export { default as ProviderBox } from './components/Provider/providerBoxVariables'
```
Comment (mnajdova) [inline packages/react/src/themes/teams-dark/componentVariables.ts]: Great that we manage to remove all dark theme overrides, just double check the screener differences.

Comment (mnajdova) [reply packages/react/src/themes/teams-dark/componentVariables.ts]: There are lots of changes in dark theme that changes the background of the Dropdown. Make sure that we are not breaking something

Comment (notandrew) [reply packages/react/src/themes/teams-dark/componentVariables.ts]: Looks good to me

**Pull request closed without merging**